test/unit/example_notebooks/test_example_notebooks.py

```diff
@@ -34,7 +35,16 @@ def test_bedrock_model_notebook(tb):
         p4.start()
         """
     )
-    tb.execute()
+
+    # Skip execution of the first code cell, which installs the *currently-released* fmeval package.
+    # We want to test the example notebooks against the latest code, not the released package.
+    seen_code_cell = False
+    for index, cell in enumerate(tb.cells):
+        if cell["cell_type"] == "code" and not seen_code_cell:
+            seen_code_cell = True
+            continue
+        super(TestbookNotebookClient, tb).execute_cell(cell, index)
```

Comment:
(non-blocking): should we extract this as a helper function, since it's being added to all the example notebook tests?

Reply:
Yeah, will update in a minute.